.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,74 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of Morpheus are you running?
+      placeholder: "example: 22.09"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installation-method
+    attributes:
+      label: Which installation methods does this occur on?
+      multiple: true
+      options:
+        - Docker
+        - Conda
+        - Source
+
+  - type: textarea
+    id: what-happened
```

Comment:
I don't love "what happened". This implies it only works for runtime issues. This really just needs to be a description and expected behavior

Reply:
Sounds good - will change wording. 

FYI the id itself doesn't show anywhere, that's just for us if we want to do anything with the graphQL api in the future.